src/components/SendModal/ProvisionIdentity/ProvisionIdentityResult/ProvisionIdentityResult.render.js

```diff
@@ -16,6 +16,7 @@ export const ProvisionIdentitySuccessRender = function () {
   const {decision} = response;
   const {result} = decision;
   const coinObj = this.props.sendModal.coinObj;
+  const formattedFriendlyName = result.fully_qualified_name.replace(".VRSC@", "@");
```

Comment:
fully_qualified_name is not a mandatory category for a provision result, so treating it as such can cause wallet crashes. The case in which identity_address is used instead must also be handled.

Reply:
This also applies to handleProvisioningResponse